front/src/components/app/index.js

```diff
@@ -13,6 +14,7 @@ class App extends Component {
                     <Route path="/student-registration/:token" component={StudentRegistrationPage} />
                     <Route path="/teachers" exact component={TeachersPage} />
                     <Route path="/login" component={LoginPage} />
+                    <Route path='/groups/create' component={GroupCreationWindow} />
```

Comment:
I think it should be a part of groups page and don't have separated route, just be a modal

Reply:
done